shared/src/common/types/user-details/user-details-item-response-dto.type.ts

```diff
@@ -0,0 +1,11 @@
+type UserDetailsItemResponseDto = {
+  avatarUrl: string | null;
+  createdAt: string;
+  dateOfBirth: string | null;
+  fullName: string;
+  gender: string | null;
```

Comment:
I think gender could be typed by a `UserGender` as it's value comes from this enum
```suggestion
  gender: UserGender | null;
```